tests/x11/shutdown.pm

```diff
@@ -39,10 +39,12 @@ sub test_flags {
 
 sub post_fail_hook {
     my ($self) = @_;
-    # In case plymouth splash shows up and the shutdown is blocked, show
-    # console logs - save screen of console (plymouth splash screen in disabled at boottime)
-    send_key('esc') if $self->{await_shutdown};
+    # Reveal what is behind Plymouth splash screen
+    wait_screen_change { send_key('esc') } if $self->{await_shutdown};
```

Comment:
sorry I still see a connection to `lib/opensusebasetest`. So far it only presses the 'esc' key but that should be done at a common place then. Also, please see other modules called "shutdown", e.g. tests/shutdown/shutdown.pm which already duplicates the 'esc' key. Please use module inheritance.

I think the `post_fail_hook` in lib/consoletest.pm can serve as a good example